src/Microsoft.AspNetCore.Server.Kestrel/Http/MessageBody.cs

```diff
@@ -178,6 +178,7 @@ public override ValueTask<int> ReadAsyncImplementation(ArraySegment<byte> buffer
         private class ForChunkedEncoding : MessageBody
         {
             private int _inputLength;
+            private int _chunkSize;
```

Comment:
Out of curiosity, is there a reason for making this a field instead of a local variable? It seems that it's still only being used when `_mode == Mode.ChunkPrefix`